rllib/core/models/tests/test_cnn_encoders.py

```diff
@@ -84,10 +74,41 @@ def test_cnn_encoders(self):
             for fw in framework_iterator(frameworks=("tf2", "torch")):
                 # Add this framework version of the model to our checker.
                 outputs = model_checker.add(framework=fw)
-                self.assertEqual(outputs[ENCODER_OUT].shape, (1, output_dims[0]))
+                # Confirm that the config conputed the correct (actual) output dims.
+                self.assertEqual(outputs[ENCODER_OUT].shape, (1, config.output_dims[0]))
 
             # Check all added models against each other.
-            model_checker.check()
+            model_checker.check(atol=0.0005)
```

Comment:
Just out of curiosity, why do we introduce tolerance?

Reply:
It's a tricky thing: I think by removing the last dense, we somehow made this test more flakey. Remember that we set ALL the weights in a network to the exact same random value (float between -0.01 and 0.01) in order to better compare tf and torch computations and I think the CNN operations might be slightly different. Not sure exactly.

But to be honest: 0.0005 absolute tolerance is totally fine with me here.